test/projects/token-wrapper/scripts/hooks.js

```diff
@@ -39,8 +56,7 @@ async function preInit({ _experimentalAppInstaller, log }, bre) {
 
   // Mint tokens.
   // NOTE: Tokens are automatically minted to the deploying address in this sample ERC20,
-  // so no need to mint to the first account. All we need to do is transfer from the first
-  // account to the second account.
+  // so no need to mint to the first account. All we need to do is transfer from the first account to the second account.
```

Comment:
Not sure on the line widths generally used, but it's generally 80-100 so we might as well keep the extra line here?